types/composite.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
You forgot to implement forward-engineering and reverse-engineering for this type

Reply:
Just to resolve the thread: RE and FE are implemented, this type was removed by mistake